snack-or-booze/src/App.js

```diff
@@ -1,51 +1,87 @@
 import React, { useState, useEffect } from "react";
-import { BrowserRouter } from "react-router-dom";
 import "./App.css";
 import Home from "./Home";
 import SnackOrBoozeApi from "./Api";
 import NavBar from "./NavBar";
 import { Route, Switch } from "react-router-dom";
-import Menu from "./FoodMenu";
-import Snack from "./FoodItem";
+import Menu from "./Menu";
+import MenuItem from "./MenuItem";
+import AddItem from './AddItem'
 
 function App() {
+  const INIITIAL_FORM_DATA = {
+    name: '',
+    description: '',
+    recipe: '',
+    serve: ''
+  }
   const [isLoading, setIsLoading] = useState(true);
-  const [snacks, setSnacks] = useState([]);
+  const [items, setItems] = useState({});
+  const [formData, setFormData] = useState(INIITIAL_FORM_DATA)
 
   useEffect(() => {
-    async function getSnacks() {
-      let snacks = await SnackOrBoozeApi.getSnacks();
-      setSnacks(snacks);
+    async function getItems() {
+      let items = await SnackOrBoozeApi.getItems();
+      setItems(items);
       setIsLoading(false);
     }
-    getSnacks();
+    getItems();
   }, []);
 
+  const changeHandler = (e) => {
+    const { name, value } = e.target;
+    setFormData((fdata) => ({
+      ...fdata,
+      [name]: value
+    }));
+  }
+
+  const submitHandler = (e, type) => {
+    e.preventDefault();
+    const newItem = {
+      ...formData,
+      id: formData.name.toLowerCase().replace(/\s/g, '-')
+    }
+    const added = SnackOrBoozeApi.addNewItem(type, newItem);
+    added ? window.location.href = `/${type}` : alert('Something went wrong. Please try again.')
```

Comment:
Should you use `history.push()` instead?
```suggestion
    added ? history.push(`/${type}`) : alert('Something went wrong. Please try again.')
```